docs/source/devinstall.md

```diff
@@ -83,7 +83,7 @@ make bdist
 Enterprise Gateway includes two sets of kernelspecs for each of the three primary kernels: `IPython`,`IR`, 
 and `Toree` to demonstrate remote kernels and their corresponding launchers.  One set uses the 
 `DistributedProcessProxy` while the other uses  the `YarnClusterProcessProxy`. The following makefile 
-target produces a tar file (`enterprise_gateway_kernelspecs.tar.gz`) in the `dist` directory.
+target produces a tar file (`enterprise_gateway_kernelspecs.tar.gz`) in the `dist` directory. (We are using `sbt` for the build so you might need to install/upgrade it.)
```

Comment:
I'd rather not use the parenthesis and be a little more specific.  How about something along these lines (sans italics) *after* the `make kernelspecs` entry ...

_"Note: Because the scala launcher requires a jar file, `make kernelspecs` requires the use of `sbt` to build the scala launcher jar.  Please consult the [sbt site](http://www.scala-sbt.org/) for directions to install/upgrade `sbt` on your platform.  We currently prefer the use of `1.0.3`."_

Reply:
Much better